idc_index/index.py

```diff
@@ -49,11 +50,24 @@ def __init__(self):
         )
 
         # Lookup s5cmd
-        self.s5cmdPath = shutil.which("s5cmd")
+        self.s5cmdPath = None
+
+        logger.debug("Checking if s5cmd is available in the package folder")
+        self.s5cmdPath = str(
+            next(Path(os.path.join(current_dir, "s5cmd")).glob("s5cmd*"), None)
+        )
+
+        if self.s5cmdPath is None:
+            logger.debug("Falling back to system s5cmd")
```

Comment:
To further clarify, the `s5cmd` tool is installed as a "script" and expected to be available in the path once the wheel is installed:

https://github.com/ImagingDataCommons/idc-index/blob/fcedf4a9bac3d0190bd5558802f031a904ddb2cc/pyproject.toml#L64-L65